packages/vhd-lib/openVhd.js

```diff
@@ -9,7 +9,7 @@ exports.openVhd = async function openVhd(handler, path, opts) {
   try {
     return await VhdFile.open(handler, resolved, opts)
   } catch (e) {
-    if (e.code !== 'EISDIR') {
+    if (e.code !== 'EISDIR' && e.code !== 'ERR_ASSERTION') {
```

Comment:
Why?

Reply:
on en encrypted remote, we will try to open VhdFile, it will fail with an ERR_ASSERTION  "remote is encrypted" and then try to open it as a VhdDirectory

Reply:
This is not obvious and warrants a comment, is this one correct?

```suggestion
    // if the remote is encrypted, trying to open a VhdFile will throw an assertion error before checking if the path is a directory, therefore we should try to open a VhdDirectory anyway.
    if (e.code !== 'EISDIR' && e.code !== 'ERR_ASSERTION') {
```